documentation/modules/proc-manual-restart-mirrormaker2-connector.adoc

```diff
@@ -14,17 +13,32 @@ This procedure describes how to manually trigger a restart of a Kafka MirrorMake
 .Procedure
 
 . Find the name of the `KafkaMirrorMaker2` custom resource that controls the Kafka MirrorMaker 2.0 connector you want to restart:
-[source,shell,subs=+quotes]
++
+[source,shell,subs="+quotes"]
+----
 kubectl get KafkaMirrorMaker2
+----
 
 . Find the name of the Kafka MirrorMaker 2.0 connector to be restarted from the `KafkaMirrorMaker2` custom resource.
-[source,shell,subs=+quotes]
-kubectl describe KafkaMirrorMaker2 _KafkaMirrorMaker2-name_
-
-. To restart the connector, annotate the `KafkaMirrorMaker2` resource in Kubernetes. For example, using `kubectl annotate` to restart the connector named `my-source->my-target.MirrorSourceConnector`:
-[source,shell,subs=+quotes]
-kubectl annotate KafkaMirrorMaker2 _KafkaMirrorMaker2-name_ "strimzi.io/restart-connector=my-source->my-target.MirrorSourceConnector"
++
+[source,shell,subs="+quotes"]
+----
+kubectl describe KafkaMirrorMaker2 _KAFKAMIRRORMAKER-2-NAME_
```

Comment:
Would it be helpful to show the output of this command?

Reply:
I don't think that is necessary.